### PR TITLE
Move supervisor pin to 3.3.1-1+deb9u1

### PR DESCRIPTION
**Pull request by mst-ableton**
Looks like the underlying Docker container has an updated version now.

@ala-ableton, @eta-ableton, @nre-ableton, @rco-ableton ptal!

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 FROM python:2-slim
 
 RUN apt-get update \
- && apt-get install -y supervisor=3.0r1-1+deb8u1 --no-install-recommends \
+ && apt-get install -y supervisor --no-install-recommends \
```
Comment (nre-ableton) [inline Dockerfile]: Hadolint is going to freak about about this. Will we suppress the warning?

**Comment by mst-ableton** [comment]
@nre-ableton I was trying to remember why I pinned it in the first place! Anyway, updated.

**Comment by ala-ableton** [comment]
LGTM :+1:

**Comment by mst-ableton** [comment]
I have no idea why Travis isn't reporting back, but the successful build is here: https://travis-ci.org/AbletonDevTools/jenkins_exporter/builds/389620331

**Pull request merged**